src/jbmc/jbmc_parse_options.cpp

```diff
@@ -524,6 +519,25 @@ int jbmc_parse_optionst::doit()
     get_message_handler(),
     prop_conv);
 
+  // unwinds <clinit> loops to number of enum elements
+  if(options.get_bool_option("java-unwind-enum-static"))
+  {
+    bmc.add_loop_unwind_handler(
+      [&goto_model]
+      // NOLINTNEXTLINE
```

Comment:
That linter problem isn't actually in the next line, so the comment needs to move to silence the linter...